server/routes/compiler.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router = express.Router();
+const axios = require('axios');
+
+router.post('/:language', async (req, res, next) => {
```

Comment:
typically it's a good idea to have some validation on the server side as well. Maybe just a simple check to make sure the language is one of the accepted ones

Reply:
Validation has been added